src/docker-compose.yml

```diff
@@ -88,5 +96,22 @@ services:
       MessageCount: 0 # send unlimited
       Template: '{"deviceId": "$.DeviceId", "deviceTimestamp": "$.Time", "doubleValue": $.DoubleValue}'
       Variables: '[{"name": "DoubleValue", "randomDouble":true, "min":0.22, "max":1.25}]'
+  prometheus:
+    image: prom/prometheus
+    volumes:
+      - ./prometheus/:/etc/prometheus/
+    command:
+      - "--config.file=/etc/prometheus/prometheus.yml"
+      - "--storage.tsdb.path=/prometheus"
+    ports:
+      - 9090:9090
```

Comment:
I exposed it as at the beginning I was accessing Prometheus UI too. It can nicely show if targets are configured correctly
![image](https://user-images.githubusercontent.com/25376553/228206298-94a3d8c6-4b3a-47bc-9bf4-83be6f154a77.png)
But yeah, I can now remove it.

Reply:
As agreed, since Prometheus UI is also useful for debugging, let's keep it exposed.